### PR TITLE
[CD] Fix the name of the pip wheels in CD

### PR DESCRIPTION
**Pull request by Zha0q1**
Now when we trigger a cd release build we get pip wheels with "REALEASE" attached to the file name. e.g.  `mxnet-2.0.0RELEASE-py3-none-manylinux2014_x86_64.whl`

Refer to https://dist.mxnet.io/python

### CODE REVIEW

File: tools/pip/setup.py [hunk 1]
```diff
@@ -58,6 +58,10 @@
 elif travis_tag.startswith('patch-'):
     __version__ = os.environ['TRAVIS_TAG'].split('-')[1]
 
+# release tag
+elif is_release:
+    __version__ += 'RELEASE'
```
Comment (leezu) [inline tools/pip/setup.py:63]: "mxnet-2.0.0RELEASE-py3-none-manylinux2014_x86_64.whl" may not be the correct naming scheme for uploading to Pypi. Also if we install mxnet from pypi and call `mxnet.__version__`, seeing `2.0.0.alphaRELEASE` may be unexpected. Do you think we should generate the wheels to be ready for upload?

**Review by leezu — COMMENTED** [review]
Thank you!

File: tools/pip/setup.py (hunk 1 above)
Comment (Zha0q1) [reply tools/pip/setup.py:63]: @leezu thanks for reviewing! I thought we would manually rename the file. In this case I can simply remove the "RELEASE" and then `mxnet-2.0.0-py3-none-manylinux2014_x86_64.whl` should be compliant I think?

**Review by leezu — APPROVED** [review]

File: ci/build.py [hunk 2]
```diff
@@ -270,7 +277,7 @@ def main() -> int:
     command = list(chain.from_iterable(args.command))
     environment = dict([(e.split('=')[:2] if '=' in e else (e, os.environ[e]))
                         for e in args.environment])
-
+    print(environment)
```
Comment (szha) [inline ci/build.py:280]: remove

**Pull request merged**